package.json

```diff
@@ -39,7 +39,7 @@
     "localforage-observable": "2.0.0",
     "mark.js": "8.11.1",
     "marked": "github:mattermost/marked#3a9d0a5bd6d735df1a716ab50f73d58a1d865bdb",
-    "mattermost-redux": "github:mattermost/mattermost-redux#0567a7736f0bfc792d29461432d474ce8daeca37",
+    "mattermost-redux": "github:mattermost/mattermost-redux#a7efaf1f476572e6f7ff5bd515cc8ab1cf88e90d",
```

Comment:
you should update this to the latest commit of the release branch

Reply:
Yeah, this is just a temporary commit from my redux PR so that QA can setup a test server. I'll update this once the redux commit is actually merged to the release branch.